utils/get_availability.py

```diff
@@ -150,21 +179,43 @@ def get_torrent_info(item, config):
         logger.error("Failed to retrieve torrent info after " + str(max_retries) + " attempts")
     logger.info("Got torrent info")
     torrent = bencode.bdecode(response.content)
-    trackers = []
+
+    #To avoid duplicates, lets use a set
+    trackers = set()
     if 'announce-list' in torrent:
         for tracker in torrent['announce-list']:
             if len(tracker) > 0:
-                trackers.append("tracker:" + tracker[0])
+                trackers.add("tracker:" + tracker[0])
+    
     if 'announce' in torrent:
-        trackers.append("tracker:" + torrent['announce'])
+        #Some torrents have lists for these for some reason
+        if isinstance(torrent['announce'], list):
+            for tracker in torrent['announce']:
+                trackers.add("tracker:" + tracker)
+        else:
+            trackers.add("tracker:" + torrent['announce'])
+
     files = []
     if 'files' in torrent['info']:
         for file in torrent['info']['files']:
             if file['path'][-1].lower().endswith(tuple(format)):
                 files.append(file['path'][-1])
 
-    hash = hashlib.sha1(bencode.bencode(torrent['info'])).hexdigest()
```

Comment:
Hash is used on line 233 (and for rd on line 21)

Reply:
Good catch! I totally forgot about the hash variable. It should be treated like the magnet link, so If any debrid like provider responds with anything other than 200 on the instantAvailability test the base32 version should be used.